README.md

```diff
@@ -284,6 +300,169 @@ el "button" [attr "type" "button", onClick_ save] do
+bindValueOnChange :: Ref String -> Prop
+
+
+-- | Attach dynamically-changing `checked` property to an input element.
+-- | The value can still be changed by user interaction.
+-- |
+-- | Only works on input `type="checkbox"` and `type="radio"` elements.
+checkedD :: Dynamic Boolean -> Prop
+
+-- | Set up a two-way binding between the `checked` of an `<input>` element,
+-- | and the given `Ref`.
+-- |
+-- | Only works on input `type="checkbox"` and `type="radio"` elements.
+bindChecked :: Ref Boolean -> Prop
+
+```
+
+Example: 
+
+```purescript
+
+import Prelude
+import Specular.Ref (Ref, newRef)
+import Specular.Dom.Browser ((:=))
+import Specular.Dom.Element (el, attrs, bindValueOnChange)
+import Specular.Dom.Widget (emptyWidget)
+
+let description :: Ref String = newRef ""
+
+el "input" [attrs ("type" := "text")    , bindValueOnChange description] emptyWidget
+
+```
+
+
+#### A Counter example
+
+```purescript
+module Main where
+
+import Prelude
+import Effect (Effect)
+
+import Data.Functor.Contravariant (cmap)
+
+import Specular.Callback (mkCallback, triggerCallback)
+import Specular.Dom.Browser ((:=))
+import Specular.Dom.Element (attrs, class_,  el,  onClick_, text, dynText)
+import Specular.Dom.Widget (runMainWidgetInBody)
+import Specular.Ref (Ref, value, newRef, modify)
+
+
+
+main :: Effect Unit
+main = do
+  -- | Will append widget to the body
+  runMainWidgetInBody do
+    counter :: Ref Int <- newRef 0
+  
+    -- | Subtract 1 from counter value the straight forward way
+    let subtractCb = mkCallback \_ -> triggerCallback (modify counter) (add (negate 1))
+
+    -- | Add 1 to counter value using the contravariant instance
+    let addCb = cmap (\_ -> add 1) (modify counter)
+
+    el "button" [class_ "btn", attrs ("type":="button" ), onClick_ addCb ] do
+      text "+"
+
+    dynText $ show <$> value counter
+
+    el "button" [class_ "btn", attrs ("type":="button" ), onClick_ subtractCb ] do
+      text "-"
+```
+
+<p class="callout warning">Warning: examples which can be found in this repo which are using "FixFRP" are deprecated !</p>
+
+
+## Getting started using spago
+
+Initialize a repository and install purescript
+
+- `npm init`
+- `npm install --save-dev purescript@0.13.8`
+
+Add `node_modules/.bin` to path:
+- `export PATH="./node_modules/.bin:$PATH"`
+
+Initialize `spago`: 
+
+- `spago init`
+
+to check if everything is working so far:
+- `spago build`
+
+Since `Specular` is not in an official `package-set`, you will have to add it manually:
+
+```dhall
+let upstream = -- This should exist in your `packages.dhall` file
+in  upstream
+  with specular =
+  { dependencies =
+    [ "prelude"
+      , "aff"
+        , "typelevel-prelude"
+        , "record"
+        , "unsafe-reference"
+        , "random"
+        , "generics-rep"
+        , "debug"
+        , "foreign-object"
+        , "contravariant"
+        , "avar"
+        ,"psci-support"
+        ,"console"
+        ,"spec"
+        ,"foreign"
```

Comment:
These are Specular's devDependencies (added to the main list, since there's no such thing in `psc-package`).

Reply:
Will do.